packages/remirror__react-renderer/src/renderer.tsx

```diff
@@ -35,9 +35,10 @@ const defaultTypeMap: MarkMap = {
   codeBlock: CodeBlock,
   orderedList: 'ol',
   text: TextHandler,
+  li: 'li'
```

Comment:
I guess The key here should be `listItem`?

Reply:
Of course you’re right 🤦